layers/enums.go

```diff
@@ -93,7 +93,7 @@ const (
 type LinkType uint8
 
 const (
-	// According to pcap-linktype(7) and http://www.tcpdump.org/linktypes.html
+	// According to pcap-linktype(7) and http://www.tcpdump.org/linktypes.html with fixes from pcap/bpf.h
```

Comment:
It is worth. On BSD it is 12 but on OpenBSD it is 14. On OpenBSD 12 is used for something else.